tests/test_cli.py

```diff
@@ -94,7 +94,7 @@ def test_normal_computation(tasmin_series, tasmax_series, pr_series, tmp_path, i
 
     ds.to_netcdf(input_file, engine="h5netcdf")
 
-    args = ["-i", str(input_file), "-o", str(output_file), "-v", indicator]
+    args = ["-i", str(input_file), "-o", str(output_file), "-v", "--engine", "h5netcdf", indicator]
```

Comment:
We could offer the `engine` as an option?

Would this technically work with an OPeNDAP URL? I know that `h5netcdf` doesn't support those. Otherwise, `h5netcdf` is a core dep of `xclim` so this is a good idea.

Reply:
I think it simply won't work. h5netcdf doesn't support OpenDAP. For users wanting to use OpenDAP the easiest would be not to pass `--engine` and let xarray decide, as usual. I added these options there only so that the CI doesn't use netCDF4, in a try to avoid crashes.